code/__defines/species_languages.dm

```diff
@@ -35,7 +35,8 @@
 #define DAIRY		0x80	// Lactose intolerance, ho! Also bad for skrell.
 #define FUNGI		0x100	// Delicious shrooms.
 #define COFFEE		0x200	// Mostly here for tajara.
-#define GENERIC		0x400	// Catchall for stuff that doesn't fall into the groups above. You shouldn't be allergic to this type, ever.
+#define EGGS		0x400	// For skrell. Just eggs.
+#define GENERIC		0x800	// Catchall for stuff that doesn't fall into the groups above. You shouldn't be allergic to this type, ever.
```

Comment:
#7940 is adding sugars as 0x800 and leaving generic as 0x400. I recommend just adding eggs at 0x1000 or else there's probably going to be conflicts.